src/models/Goal.ts

```diff
@@ -1,4 +1,4 @@
-import mongoose, { Schema } from "mongoose"
+import mongoose, { Schema } from 'mongoose'
```

Comment:
Remove file. Work related to Goals will be done in separate branches per crud operation